src/serac/physics/solid_mechanics.hpp

```diff
@@ -137,10 +137,11 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
   SolidMechanics(const NonlinearSolverOptions nonlinear_opts, const LinearSolverOptions lin_opts,
                  const serac::TimesteppingOptions timestepping_opts, const GeometricNonlinearities geom_nonlin,
                  const std::string& physics_name, std::string mesh_tag, std::vector<std::string> parameter_names = {},
-                 int cycle = 0, double time = 0.0, bool checkpoint_to_disk = false)
+                 int cycle = 0, double time = 0.0, bool checkpoint_to_disk = false, bool use_warm_start = true)
```

Comment:
This needs to be documented... its documented below with this:

```
   * @param use_warm_start A flag to turn on or off the displacement warm start predictor which helps robustness for
   * large deformation problems
```
I would submit a code suggestion but its too far away from a changed line =/

Reply:
Done!